primitives/linux-port/column.cpp

```diff
@@ -1694,10 +1975,10 @@ void PrimitiveProcessor::scanAndFilterTypeDispatcher(NewColRequestHeader* in, Co
     const uint16_t ridSize = in->NVALS;
     uint16_t* ridArray = in->getRIDArrayPtr(W);
     const uint32_t itemsPerBlock = logicalBlockMode ? BLOCK_SIZE : BLOCK_SIZE / W;
-    filterColumnData<T, KIND_FLOAT>(in, out, ridArray, ridSize, block, itemsPerBlock, parsedColumnFilter);
+    filterColumnData<T, KIND_FLOAT>(in, out, ridArray, ridSize, block, itemsPerBlock, parsedColumnFilter, hasAuxCol, blockAux);
     return;
   }
-  _scanAndFilterTypeDispatcher<T>(in, out);
+  _scanAndFilterTypeDispatcher<T>(in, out, hasAuxCol);
```

Comment:
What is the semantics of hasAuxCol exactly? If it is true when we process the aux column for a table or it is true for ColumnCommand for any column of that table?

Reply:
Could you put this variable into in message plz?

Reply:
Yes I agree, `hasAuxCol` does not belong here as an additional function argument. So I have moved it inside the `in` message.